modules/@apostrophecms/ui/ui/apos/components/AposSelect.vue

```diff
@@ -55,7 +55,7 @@ export default {
       }
     },
     selected: {
-      type: [ String, Number ],
+      type: [ String, Number, Boolean ],
```

Comment:
Can you remove this restriction by type entirely? The point here is to accept any JSON-compatible value, even `null` or an object or array.

Reply:
I would recommend a comment so no one jumps in to add it back.

Reply:
We have to let the type verification (eslint rule), which makes it hard to decide what to accept.

We could allow [all the types](https://vuejs.org/guide/components/props.html#prop-validation) but I don't see how select options would contain an array or an object...

If null or undefined is passed, then it will take the default `''` value.

I think that String, Number and Boolean make sense for a selected  value